.eslintrc.js

```diff
@@ -62,7 +62,7 @@ module.exports = {
     ],
     "linebreak-style": [
       "error",
-      "unix"
+      "windows"
```

Comment:
Instead of doing this, please update your gitattribute file based on the info in the link below
https://eslint.org/docs/rules/linebreak-style#using-this-rule-with-version-control-systems